spec/controllers/home_controller_spec.rb

```diff
@@ -3,8 +3,8 @@
 RSpec.describe HomeController, type: :controller do
   describe 'GET #index' do
     before(:each) do
-      stub_request(:get, /redmine.renuo.ch/)
-        .to_return(status: 200, body: 'Test', headers: {})
+      http_basic_auth
+      stub_request(:get, /redmine.renuo.ch/).to_return(status: 200, body: 'Test', headers: {})
     end
```

Comment:
Do you have an idea of how this could be done better? Is there a test missing?

<sub>sʇxəʇuoɔ ɔədsɹ :ʇuıɥ</sub>